CSSTooltipPseudo/explainer.md

```diff
@@ -0,0 +1,896 @@
+getting accessibility correct quite difficult.
+
+As such, this solution requires a lot of work to get right, especially if all they
+want to do is adjust a few simple styles on the built-in tooltip.
+
+#### Utilize `popover`, CSS anchor positioning, and/or `interesttarget`
+
+Although creating a custom tooltip can be cumbersome for an author who is looking
+to just adjust a few simple styles, they do have some new web APIs, and APIs that
+are in the works, that can help them in accomplishing a fully customizable tooltip
+more easily:
+  - The [`popover`](https://html.spec.whatwg.org/multipage/popover.html#dom-popover) 
+    attribute, which is baseline across browsers as of April 2024.
+  - [CSS Anchor Positioning](https://drafts.csswg.org/css-anchor-position-1/),
+    which is not yet baseline, but is part of the [Interop 2025 effort](
+    https://github.com/web-platform-tests/interop/blob/main/2025/README.md#css-anchor-positioning).
+  - The [`interesttarget`](
+    https://open-ui.org/components/interest-invokers.explainer/#keyboard) attribute,
+    which is currently being investigated, with a prototype already in working
+    shape in Chromium.
+
+One benefit that authors get with `interesttarget` over the other existing solutions
+is that it creates a more accessible experience out-of-the-box and handles input
+handling for the author, creating a more seamless approach than is available with
+the `popover`attribute on its own. It is also a great solution if an author wants
+custom HTML within their tooltip, as opposed to simple textual content.
+
+However, `interesttarget` does have some limitations. Currently, `interesttarget`
+is only allowed on buttons and links, which is more limited than the `title`
+attribute.
+_____________________________
+
+All of these options do allow authors to create more custom tooltip experiences
+than what is currently available through the browser's built-in option using the
+`title` attribute, but these options often require more work than should be
+necessary for a simple style adjustment.
+
+## Current landscape of built-in tootlips
+
+The rendering of tooltips via the `title` attribute is up to each UA. Below
+is a brief survey of how some of the top browser engines render built-in
+tooltips today, in no particular order.
+
+### Simple tooltips across browsers
+
+In this section is a comparison of basic tooltip rendering across each
+browser engine using the code example below.
+
+```html
+<button title="This is a button!">Button Example</button> 
+```
+
+#### Chromium
+
+In Chromium, the tooltip is rendered above all web content, and the position
+of the tooltip is dependent on the position of the mouse and the element it
+is anchored to.
+
+The style of the tooltip that is rendered is very simple: a white or black
+background with black or white text and a matching border.
+
+On Windows the simple test case above renders as follows:
+
+<p align="center">
```

Comment:
These images don't show up for me. Is this just something to do with the PR, and they'll work once it's landed?

Should they be using markdown image syntax instead of HTML? https://www.digitalocean.com/community/tutorials/markdown-markdown-images

Also, these should have alt text 🤓

Reply:
I had looked at the Gap Decorations explainer as an example for images. But updated to the markdown version (which also means I wasn't able to center them, which isn't a problem).

Also added alt text.

The images show up for me locally, so I think this is an issue with the Github tooling. But this happened with other explainers I reviewed, so they should show up correctly once this lands.